### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Matthww**
This fixes the bugs I came across

image

### CODE REVIEW

**Comment by Thunder33345** [comment]
yeah what bugs doe?

**Comment by Matthww** [comment]
@Thunder33345 Namespace errors, use statement errors and some more

File: src/xenialdan/MapAPI/Commands.php [hunk 1]
```diff
@@ -6,8 +6,8 @@
 use pocketmine\utils\TextFormat;
 use pocketmine\Player;
 use pocketmine\plugin\Plugin;
-use xenialdan\MapUtils\subcommand\LoadSubCommand;
-use xenialdan\MapUtils\subcommand\SubCommand;
+use xenialdan\MapAPI\subcommand\LoadSubCommand;
+use xenialdan\MapAPI\subcommand\SubCommand;
```
Comment (inxomnyaa) [inline src/xenialdan/MapAPI/Commands.php:10]: damn indeed.

**Comment by inxomnyaa** [comment]
please change Color back ;) othervise.. yea i was pretty tired, slipped through

did you generate that pic with this plugin? i got no time to test 😂

**Comment by Matthww** [comment]
I did generate it with this plugin, I will change Color back it was just temp

**Comment by inxomnyaa** [comment]
Wait a sec and i'll push some local shit ;)

**Comment by MisteFr** [comment]
@Matthww After a reboot, if you have placed the map in an ItemFrame the map is still visible ? Or she becames invisible ?

**Comment by inxomnyaa** [comment]
"she" will be invisible. thats why i am pushing local work now

**Comment by Matthww** [comment]
I can close this?

**Comment by inxomnyaa** [comment]
close or rebase on changes, idk..

**Pull request closed without merging**